src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -651,8 +653,7 @@ private void ScanInterpolatedStringLiteralHoleBracketed(char start, char end)
             {
                 Debug.Assert(start == _lexer.TextWindow.PeekChar());
                 _lexer.TextWindow.AdvanceChar();
-                int colon = 0;
-                ScanInterpolatedStringLiteralHoleBalancedText(end, isHole: false, ref colon);
+                ScanInterpolatedStringLiteralHoleBalancedText(end, isHole: false, out _);
```

Comment:
nit: consider naming the discarded argument

Reply:
sure :)